docs/kratos/concepts/terminology.mdx

```diff
@@ -1,23 +0,0 @@
----
-id: terminology
-title: Terminology
----
-
-The purpose of this document is to define terminology used through this documentation.
-
-## End-User
-
-The person who actually uses a particular product that uses Ory Kratos.
-
-## Identity
-
-the "who" of a software system. It can be a customer, employee, user, contractor, or even a programmatic identity such as an IoT
-device, application, or some other type of "robot."
-
-Common aliases are "user", "user account", "account", "subject".
-
-## Self-Service
```

Comment:
I struggled understanding "self-service" the first time I read it as a non-native english speaker. Should we explain this somewhere? And if so, where?

Reply:
Self-service flows are described here: 
https://github.com/ory/docs/blob/31e15236b390bc8fd2244109d44fb627063711aa/docs/concepts/terminology.mdx

But there's no definition of what "self-service" is in general. I can add it there. 

This change was made to have a  single, overarching "concepts" doc instead of two separate ones.

Reply:
Got it, makes total sense